### PR TITLE
Master

### PR DESCRIPTION
**Pull request by ZillDing**
a) Use cross-env in npm scripts
b) Fix resizable class name

### CODE REVIEW

File: examples/1.html [hunk 1]
```diff
@@ -10,7 +10,6 @@
         background: #eee;
       }
       .box {
-        position: relative;
```
Comment (STRML) [inline examples/1.html]: Why was this removed?

Comment (ZillDing) [inline examples/1.html]: Thanks for quick reply. This style already exists in `css/styles.css`. I think this is redundant and may cause confusion since in order to make the resizable component work, the parent position has to be 'relative'.

Comment (STRML) [inline examples/1.html]: Gotcha. Thanks for the catch.
